mdpow/tests/test_equilibration_script.py

```diff
@@ -0,0 +1,39 @@
+import tempdir as td
+import os
+import manifest
+
+from mdpow.run import equilibrium_simulation
+from mdpow.config import get_configuration
+
+class TestEquilibriumScript(object):
+    
+    def setup(self):
+        self.tmpdir = td.TempDir()
+        self.old_path = os.getcwd()
+        self.resources = self.old_path + "/mdpow/tests/testing_resources"
+        m = manifest.Manifest(os.path.join(self.resources,'manifest.yml'))
+        m.assemble('base',self.tmpdir.name)
+
+    def _run_equil(self, solvent, dirname):
+        try:
+            print(os.listdir('.'))
+            cfg = get_configuration('runinput.yml')
+            self.S = equilibrium_simulation(cfg, solvent, dirname=dirname)
+        except:
+            assert False
```

Comment:
This tests if the simulation can be run. Ok for a start but we eventually want to test for regression and correctness. Add a comment.

Also let the assertion say something like
```python
raise AssertionError("equilibrium_simulation() failed to run")
```